doc-content/enterprise-only/getting-started-kogito/con-kogito-operator-deployment-configs.adoc

```diff
@@ -0,0 +1,23 @@
+[id="con-kogito-operator-deployment-configs_{context}"]
+= {KOGITO} service properties configuration
+
+When a {KOGITO} service is deployed, a `configMap` resource is created for the `application.properties` configuration of the {KOGITO} service.
```

Comment:
In product, we call it microservice
```suggestion
When a {KOGITO} microservice is deployed, a `configMap` resource is created for the `application.properties` configuration of the {KOGITO} microservice.
```

Reply:
Done,and applying change to the rest of the PR